Mathlib/RingTheory/OreLocalization/Basic.lean

```diff
@@ -355,91 +389,100 @@ protected theorem mul_smul (x y : R[S⁻¹]) (z : X[S⁻¹]) : (x * y) • z = x
   symm; apply oreDiv_smul_char
   rw [Submonoid.coe_mul, Submonoid.coe_mul, ← mul_assoc, ← hc, mul_assoc _ ra, ← ha, mul_assoc]
 
+@[to_additive]
 protected theorem mul_assoc (x y z : R[S⁻¹]) : x * y * z = x * (y * z) :=
   OreLocalization.mul_smul x y z
 #align ore_localization.mul_assoc OreLocalization.mul_assoc
 
+@[to_additive]
 instance instMonoidOreLocalization : Monoid R[S⁻¹] :=
   { OreLocalization.instMulOreLocalization,
     OreLocalization.instOneOreLocalization with
     one_mul := OreLocalization.one_mul
     mul_one := OreLocalization.mul_one
     mul_assoc := OreLocalization.mul_assoc }
 
+@[to_additive]
```

Comment:
```suggestion
@[to_additive  AddOreLocalization.instAddActionAddOreLocalization]
```

Reply:
I think the solution to these is maybe to not name the instance and the to-additive version and let lean handle the names, as the generated names seem good enough and we will almost never refer to this.

Reply:
Yes, in the end I agree. Please commit whatever you prefer and leave the rest. I will then be happy to call the maintainers for the merge, just ping me.